lib/ui/profitloss/RecordAmountListItem.dart

```diff
@@ -0,0 +1,401 @@
+    Color actionItemBackgroundColor,
+    TextStyle titleTextStyle,
+    TextStyle pendingDetailTextStyle,
+    TextStyle detailTextStyle,
+    EdgeInsets actionItemEdgePadding,
+    EdgeInsets cardMargins,
+    CrossAxisAlignment itemAlignment,
+    double actionItemElevation,
+    double iconHeight,
+    double iconLineSpace,
+    double detailTextSpacing,
+    int maxLines,
+  }) {
+    return RecordAmountListItemStyle(
+      actionItemBackgroundColor:
+          actionItemBackgroundColor ?? this.actionItemBackgroundColor,
+      titleTextStyle: titleTextStyle ?? this.titleTextStyle,
+      pendingDetailTextStyle:
+          pendingDetailTextStyle ?? this.pendingDetailTextStyle,
+      detailTextStyle: detailTextStyle ?? this.detailTextStyle,
+      actionItemEdgePadding:
+          actionItemEdgePadding ?? this.actionItemEdgePadding,
+      cardMargins: cardMargins ?? this.cardMargins,
+      itemAlignment: itemAlignment ?? this.itemAlignment,
+      actionItemElevation: actionItemElevation ?? this.actionItemElevation,
+      iconHeight: iconHeight ?? this.iconHeight,
+      iconLineSpace: iconLineSpace ?? this.iconLineSpace,
+      detailTextSpacing: detailTextSpacing ?? this.detailTextSpacing,
+      maxLines: maxLines ?? this.maxLines,
+    );
+  }
+}
+
+class _DefaultStyle extends RecordAmountListItemStyle {
+  final Color actionItemBackgroundColor = const Color(0x00000000);
+
+  final TextStyle titleTextStyle = const TextStyle(
+    fontSize: 17,
+    fontWeight: FontWeight.w400,
+    color: Color(0xFF1a1b46),
+  );
+  final TextStyle pendingDetailTextStyle = const TextStyle(
+    fontSize: 15,
+    fontWeight: FontWeight.normal,
+    color: Color(0x4c767690),
+  );
+  final TextStyle detailTextStyle = const TextStyle(
+    fontSize: 15,
+    fontWeight: FontWeight.normal,
+    color: Color(0xff767690),
+  );
+
+  final EdgeInsets actionItemEdgePadding =
+      const EdgeInsets.only(left: 32, right: 32, top: 25.8, bottom: 25.8);
+  final EdgeInsets cardMargins = const EdgeInsets.all(0);
+
+  final double actionItemElevation = 0;
+  final CrossAxisAlignment itemAlignment = CrossAxisAlignment.center;
+
+  final double iconHeight = 20;
+  final double iconLineSpace = 22;
+  final double detailTextSpacing = 13;
+
+  final int maxLines = 5;
+
+  const _DefaultStyle(
+      {Color actionItemBackgroundColor,
+      TextStyle titleTextStyle,
+      TextStyle pendingDetailTextStyle,
+      TextStyle detailTextStyle,
+      EdgeInsets actionItemEdgePadding,
+      EdgeInsets cardMargins,
+      CrossAxisAlignment itemAlignment,
+      double actionItemElevation,
+      double iconHeight,
+      double iconLineSpace,
+      double detailTextSpacing,
+      int maxLines});
+}
+
+const RecordAmountListItemStyle _defaultStyle = const _DefaultStyle();
+
+class RecordAmountListItem extends StatefulWidget {
+  final RecordAmountListItemStyle _style;
+  final RecordAmountListItemViewModel _viewModel;
+  RecordAmountState parent;
```

Comment:
What happens if we want to use this widget and we don't have a `RecordAmount` widget as a parent?